pkg/slo-controller/noderesource/framework/README.md

```diff
@@ -0,0 +1,54 @@
+# NodeResource Framework
+
+## Overview
+
+The node resource controller is responsible for calculating the over-commit resources and updating the result on the
+node. The node resource framework defines some extension points for resource calculation and updating strategy. Each
+node resource plugin can implement one or more stages to reconcile the node resources.
+
+The current extension stages provided by the node resource framework are as following:
+
+- **Setup**: It setups the plugin with options like controller client, scheme and event recorder.
+- **Calculate**: It calculates the node resources according to the Node and NodeMetric. In case of the NodeMetric is
+abnormal, a plugin can implement the degraded calculation inside this stage.
+- **Reset**: It resets the node resources when the colocation is configured as disabled.
```

Comment:
Calculate and reset seem to be the same extension point, how to understand the difference?

Reply:
Fixed. The Reset stage is removed, and the reset is considered a method of the Calculate stage.